docs/medical-api/handling-data/webhooks.mdx

```diff
@@ -366,6 +368,104 @@ The format follows:
   </Expandable>
 </ResponseField>
 
+### Bulk Patient Create
+
+During the processing of a [Bulk Patient Create](/medical-api/api-reference/patient/bulk-create-patient),
+the Metriport API sends Webhook messages of type `medical.bulk-patient-create` to indicate the status
+update and provide relevant information.
+
+You can expect to get one or two messages of this type for each Bulk Patient Create job, with the status
```

Comment:
I see here we can receive one or two. Can we be bit more clear? Why in some cases would we receive one vs 2?

Reply:
Will clarify

Reply:
done